apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useAddMember.ts

```diff
@@ -34,6 +35,7 @@ export function useAddMember({ device }: { device: Device | null }): DrawerProps
   const transitionToNextScreen = useCallback(
     (trustchainResult: TrustchainResult) => {
       dispatch(setTrustchain(trustchainResult.trustchain));
+      track("ledgersync_activated");
```

Comment:
suggestion (optional) : string based keys cannot be extracted into an enum so that it will be easy to maintain in a single plase?

Reply:
Yeah it would be nice to have these events in an enum somewhere. I created a features/Analytics in the new arch.
WDYT ?

Reply:
Hey there is already a file for this inside WalletSync feature : for LLM => useLedgerSyncAnalytics. So I guess we can add the enum in it or create an enum file in the type folder
And on LLD there is an enum file with the analytics so you can add it there !

Reply:
@LucasWerey 2 things, 

- enums in useLedgerSyncAnalytics are LedgerSync specific and are not event based , its page based. 
The idea behind this review is to extract tracking **events** strings and put them in a enums.ts file. 
- for a better readability and orgranisation , we need to separate types from enums as it's not the same thing (enums can be passed as a type but not the inverse), so `types.ts` or any `*.d.ts` should be for types and interfaces if needed and `enums.ts` for enums.